largest_sum_contiguous_subarray/largestSumContiguousSubarray.c

```diff
@@ -6,20 +6,21 @@ int largestSumContinousSubArray(int arr[], int size) {
 	int max_int = 0;
 	for (int i = 0; i < size; i++) {
 		max_int = max_int + arr[i];
-		if(max_till < max_int) {
-			max_till = max_int;
-		}
-		if (max_int < 0) {
-			max_int = 0;
-		}
+		max_int = max(max_int, arr[i]);
+		max_till = max(max_till, max_int);
 	}
 	return max_till;
 }
 
-int main() {
-	int array[10];
-	for (int k = 0; k < 10; k++) {
-		array[k] = rand() % 10;
+int max(int first, int second) {
+	if (first < second)	{
+		return second;
+	} else {
+		return first;
 	}
+}
+
+int main() {
+	int array[] = {-2, 1, -3, 4, -1, 2, 1, -5, 4, 5};
 	printf("%d", largestSumContinousSubArray(array, 10));
```

Comment:
Also, instead of `%d` use `%d\n`, as this is preferred and used in other C implementation.